### ISSUE TITLE
URGENT: Engine events are called on one node but no events are processed on another machine with same installer

### ISSUE THREAD

**Issue by mobycorp** [wixtoolset/issues#6884]
Sorry, I opened this before I read about needing to do this in a discussion thread (which I have now done...). I still think there is a bug here, but I don't know the proper questions to ask... If you agree, ask me the questions you want answers to and I'll provide them.

* Which version of WiX are you building with?

WiX 3.11.2.4516

* Which version of Visual Studio are you building with (if any)?

Visual Studio 2017 (I want to use VS2022 but, sadly, there is no .NET 6 version of the bootstrapper.. :-(

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?

WiX Toolset VS 2017 Extension, v1.0.0.18

* Which version of .NET are you building with?

.NET 4.8

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

There are two nodes at issue here: My development machine works perfectly, but when tested on a Windows 11 VM, none of the Engine events are fired.

Dev box: Windows 11 Pro 21H2 build 22000.856
VM: Windows 11 Pro build 22000.376

* Describe the problem and the steps to reproduce it.

Run the installer on my dev box for success and any other box for complete failure. This is HUGELY disappointing as I have been working on a custom BA so I can collect user registration information. As I said, this works perfectly in development, but not at all in production. I need to get this fixed URGENTLY!!

I have placed logger calls all over the place to identify what is going on. I don't see any calls being made for the OnXXX events in production. I have attached the log files from both runs. Other than the time wasted on adding more and more log messages, I don't know how else to troubleshoot this.

* Describe the behavior you expected and how it differed from the actual behavior.

As I have said, the installer works fine on my dev box, but not on my VM (which is meant to test how the installer will respond in the real world...)

Here is the log from the VM where no events are executed:
```
[1F9C:03B0][2022-08-26T13:55:11]i001: Burn v3.11.2.4516, Windows v10.0 (Build 22000: Service Pack 0), path: C:\Users\User\AppData\Local\Temp\{12ACBD08-BAE8-47FF-9735-452FEB8000BD}\.cr\MobyBundle.exe
[1F9C:03B0][2022-08-26T13:55:11]i009: Command Line: '-burn.clean.room=C:\Users\User\Desktop\MobyBundle.exe -burn.filehandle.attached=756 -burn.filehandle.self=760'
[1F9C:03B0][2022-08-26T13:55:11]i000: Setting string variable 'WixBundleOriginalSource' to value 'C:\Users\User\Desktop\MobyBundle.exe'
[1F9C:03B0][2022-08-26T13:55:11]i000: Setting string variable 'WixBundleOriginalSourceFolder' to value 'C:\Users\User\Desktop\'
[1F9C:03B0][2022-08-26T13:55:11]i000: Setting string variable 'WixBundleLog' to value 'C:\Users\User\AppData\Local\Temp\Moby_20220826135511.log'
[1F9C:03B0][2022-08-26T13:55:11]i000: Setting string variable 'WixBundleName' to value 'Moby'
[1F9C:03B0][2022-08-26T13:55:11]i000: Setting string variable 'WixBundleManufacturer' to value 'Moby Corp.'
[1F9C:03B0][2022-08-26T13:55:11]i000: Loading managed bootstrapper application.
[1F9C:03B0][2022-08-26T13:55:11]i000: Creating BA thread to run asynchronously.
[1F9C:2A38][2022-08-26T13:55:11]i000: Launching the MobyBA UX
[1F9C:03B0][2022-08-26T13:55:11]i100: Detect begin, 2 packages
[1F9C:03B0][2022-08-26T13:55:11]i000: Setting string variable 'Netfx4x64FullVersion' to value '4.8.04161'
[1F9C:03B0][2022-08-26T13:55:11]i000: Setting string variable 'Netfx4FullVersion' to value '4.8.04161'
[1F9C:2A38][2022-08-26T13:55:11]i000: In VM ctor
[1F9C:03B0][2022-08-26T13:55:11]i052: Condition 'Netfx4FullVersion AND (NOT VersionNT64 OR Netfx4x64FullVersion)' evaluates to true.
[1F9C:03B0][2022-08-26T13:55:11]i101: Detected package: Netfx4Full, state: Present, cached: None
[1F9C:03B0][2022-08-26T13:55:11]i101: Detected package: MobyInstallerId, state: Absent, cached: None
[1F9C:03B0][2022-08-26T13:55:11]i199: Detect complete, result: 0x0
[1F9C:2A38][2022-08-26T13:55:11]i000: VM ctor done
[1F9C:03B0][2022-08-26T13:55:40]i500: Shutting down, exit code: 0x0
[1F9C:03B0][2022-08-26T13:55:40]i410: Variable: Netfx4FullVersion = 4.8.04161
[1F9C:03B0][2022-08-26T13:55:40]i410: Variable: Netfx4x64FullVersion = 4.8.04161
[1F9C:03B0][2022-08-26T13:55:40]i410: Variable: VersionNT64 = 10.0.0.0
[1F9C:03B0][2022-08-26T13:55:40]i410: Variable: WixBundleAction = 5
[1F9C:03B0][2022-08-26T13:55:40]i410: Variable: WixBundleElevated = 0
[1F9C:03B0][2022-08-26T13:55:40]i410: Variable: WixBundleLog = C:\Users\User\AppData\Local\Temp\Moby_20220826135511.log
[1F9C:03B0][2022-08-26T13:55:40]i410: Variable: WixBundleManufacturer = Moby Corp.
[1F9C:03B0][2022-08-26T13:55:40]i410: Variable: WixBundleName = Moby
[1F9C:03B0][2022-08-26T13:55:40]i410: Variable: WixBundleOriginalSource = C:\Users\User\Desktop\MobyBundle.exe
[1F9C:03B0][2022-08-26T13:55:40]i410: Variable: WixBundleOriginalSourceFolder = C:\Users\User\Desktop\
[1F9C:03B0][2022-08-26T13:55:40]i410: Variable: WixBundleProviderKey = {e1df1198-db62-4c5b-9bd9-afb39d021057}
[1F9C:03B0][2022-08-26T13:55:40]i410: Variable: WixBundleSourceProcessFolder = C:\Users\User\Desktop\
[1F9C:03B0][2022-08-26T13:55:40]i410: Variable: WixBundleSourceProcessPath = C:\Users\User\Desktop\MobyBundle.exe
[1F9C:03B0][2022-08-26T13:55:40]i410: Variable: WixBundleTag = 
[1F9C:03B0][2022-08-26T13:55:40]i410: Variable: WixBundleUILevel = 4
[1F9C:03B0][2022-08-26T13:55:40]i410: Variable: WixBundleVersion = 1.0.0.0
[1F9C:03B0][2022-08-26T13:55:40]i007: Exit code: 0x0, restarting: No
```

I apologize in advance for the long log file... Here is the log file from my development box. Notice it goes all the way through the end...
```
[1633C:58E8][2022-08-26T12:42:51]i001: Burn v3.11.2.4516, Windows v10.0 (Build 22000: Service Pack 0), path: C:\Users\cimfr\AppData\Local\Temp\{28B0C097-B89C-41FA-9AD0-C1EA1BBEDFB3}\.cr\MobyBundle.exe
[1633C:58E8][2022-08-26T12:42:51]i009: Command Line: '-burn.clean.room=D:\Development\Moby\Dev\MobyWindowsInstaller\MobyBundle\bin\Debug\MobyBundle.exe -burn.filehandle.attached=788 -burn.filehandle.self=608'
[1633C:58E8][2022-08-26T12:42:51]i000: Setting string variable 'WixBundleOriginalSource' to value 'D:\Development\Moby\Dev\MobyWindowsInstaller\MobyBundle\bin\Debug\MobyBundle.exe'
[1633C:58E8][2022-08-26T12:42:51]i000: Setting string variable 'WixBundleOriginalSourceFolder' to value 'D:\Development\Moby\Dev\MobyWindowsInstaller\MobyBundle\bin\Debug\'
[1633C:58E8][2022-08-26T12:42:51]i000: Setting string variable 'WixBundleLog' to value 'C:\Users\cimfr\AppData\Local\Temp\Moby_20220826124251.log'
[1633C:58E8][2022-08-26T12:42:52]i000: Setting string variable 'WixBundleName' to value 'Moby'
[1633C:58E8][2022-08-26T12:42:52]i000: Setting string variable 'WixBundleManufacturer' to value 'Moby Corp.'
[1633C:58E8][2022-08-26T12:42:52]i000: Loading managed bootstrapper application.
[1633C:58E8][2022-08-26T12:42:52]i000: Creating BA thread to run asynchronously.
[1633C:166C0][2022-08-26T12:42:52]i000: Launching the MobyBA UX
[1633C:58E8][2022-08-26T12:42:52]i100: Detect begin, 2 packages
[1633C:58E8][2022-08-26T12:42:52]i000: Setting string variable 'Netfx4x64FullVersion' to value '4.8.04161'
[1633C:58E8][2022-08-26T12:42:52]i000: Setting string variable 'Netfx4FullVersion' to value '4.8.04161'
[1633C:166C0][2022-08-26T12:42:52]i000: In VM ctor
[1633C:166C0][2022-08-26T12:42:52]i000: VM ctor done
[1633C:58E8][2022-08-26T12:42:52]i052: Condition 'Netfx4FullVersion AND (NOT VersionNT64 OR Netfx4x64FullVersion)' evaluates to true.
[1633C:58E8][2022-08-26T12:42:52]i000: Detect package completed
[1633C:58E8][2022-08-26T12:42:52]i000: PackageId: Netfx4Full
[1633C:58E8][2022-08-26T12:42:52]i000: State: Present
[1633C:58E8][2022-08-26T12:42:52]i000: Detect package completed
[1633C:58E8][2022-08-26T12:42:52]i000: PackageId: MobyInstallerId
[1633C:58E8][2022-08-26T12:42:52]i000: State: Absent
[1633C:58E8][2022-08-26T12:42:52]i000: Registration in
[1633C:58E8][2022-08-26T12:42:52]i000: Registration out
[1633C:58E8][2022-08-26T12:42:52]i101: Detected package: Netfx4Full, state: Present, cached: None
[1633C:58E8][2022-08-26T12:42:52]i101: Detected package: MobyInstallerId, state: Absent, cached: None
[1633C:58E8][2022-08-26T12:42:52]i199: Detect complete, result: 0x0
[1633C:166C0][2022-08-26T12:43:11]i000: GetUser results: NotFound
[1633C:166C0][2022-08-26T12:43:11]i000: AddUser results: OK
[1633C:166C0][2022-08-26T12:44:04]i000: VerifyRegisteration results: OK
[1633C:166C0][2022-08-26T12:44:04]i000: Setting string variable 'Action' to value 'Sign Up'
[1633C:166C0][2022-08-26T12:44:04]i000: Setting string variable 'EmailAddress' to value 'cimframe@gmail.com'
[1633C:166C0][2022-08-26T12:44:04]i000: Setting string variable 'Name' to value 'Steve Miller'
[1633C:166C0][2022-08-26T12:44:04]i000: Setting string variable 'AgentInstallationLocation' to value 'C:\Program Files (x86)\Moby\Logistics Services'
[1633C:166C0][2022-08-26T12:44:04]i000: Setting string variable 'ClientInstallationLocation' to value 'C:\Program Files (x86)\Moby\MobyClient'
[1633C:166C0][2022-08-26T12:44:04]i000: Setting string variable 'DataInstallationLocation' to value 'C:\Users\cimfr\AppData\Roaming\Moby\Client Data'
[1633C:166C0][2022-08-26T12:44:19]i000: Into Process Install
[1633C:166C0][2022-08-26T12:44:19]i000: InstallLabel: Begin Installation
[1633C:58E8][2022-08-26T12:44:19]i200: Plan begin, 2 packages, action: Install
[1633C:58E8][2022-08-26T12:44:19]w321: Skipping dependency registration on package with no dependency providers: Netfx4Full
[1633C:58E8][2022-08-26T12:44:19]i000: Setting string variable 'WixBundleRollbackLog_MobyInstallerId' to value 'C:\Users\cimfr\AppData\Local\Temp\Moby_20220826124251_000_MobyInstallerId_rollback.log'
[1633C:58E8][2022-08-26T12:44:19]i000: Setting string variable 'WixBundleLog_MobyInstallerId' to value 'C:\Users\cimfr\AppData\Local\Temp\Moby_20220826124251_000_MobyInstallerId.log'
[1633C:58E8][2022-08-26T12:44:19]i201: Planned package: Netfx4Full, state: Present, default requested: Present, ba requested: Present, execute: None, rollback: None, cache: No, uncache: No, dependency: None
[1633C:58E8][2022-08-26T12:44:19]i201: Planned package: MobyInstallerId, state: Absent, default requested: Present, ba requested: Present, execute: Install, rollback: Uninstall, cache: Yes, uncache: No, dependency: Register
[1633C:58E8][2022-08-26T12:44:19]i000: Plan phase complete - Status: 0
[1633C:58E8][2022-08-26T12:44:19]i299: Plan complete, result: 0x0
[1633C:58E8][2022-08-26T12:44:19]i300: Apply begin
[1633C:58E8][2022-08-26T12:44:19]i000: Apply phase count incremented: 2
[1633C:58E8][2022-08-26T12:44:19]i000: Apply phase beginning
[1633C:58E8][2022-08-26T12:44:19]i010: Launching elevated engine process.
[1633C:58E8][2022-08-26T12:44:19]i011: Launched elevated engine process.
[1633C:58E8][2022-08-26T12:44:19]i012: Connected to elevated engine.
[92D4:ED9C][2022-08-26T12:44:19]i358: Pausing automatic updates.
[92D4:ED9C][2022-08-26T12:44:19]i359: Paused automatic updates.
[92D4:ED9C][2022-08-26T12:44:19]i360: Creating a system restore point.
[92D4:ED9C][2022-08-26T12:44:29]i361: Created a system restore point.
[92D4:ED9C][2022-08-26T12:44:29]i370: Session begin, registration key: SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{e1df1198-db62-4c5b-9bd9-afb39d021057}, options: 0x7, disable resume: No
[92D4:ED9C][2022-08-26T12:44:30]i000: Caching bundle from: 'C:\Users\cimfr\AppData\Local\Temp\{7468DA97-2C3F-48ED-A69B-647C74457BC1}\.be\MobyBundle.exe' to: 'C:\ProgramData\Package Cache\{e1df1198-db62-4c5b-9bd9-afb39d021057}\MobyBundle.exe'
[92D4:ED9C][2022-08-26T12:44:30]i320: Registering bundle dependency provider: {e1df1198-db62-4c5b-9bd9-afb39d021057}, version: 1.0.0.0
[92D4:ED9C][2022-08-26T12:44:30]i371: Updating session, registration key: SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{e1df1198-db62-4c5b-9bd9-afb39d021057}, resume: Active, restart initiated: No, disable resume: No
[92D4:11DF0][2022-08-26T12:44:30]i305: Verified acquired payload: MobyInstallerId at path: C:\ProgramData\Package Cache\.unverified\MobyInstallerId, moving to: C:\ProgramData\Package Cache\{1E243719-FDEC-432D-8279-ACE4825D1F7D}v1.0.0.0\MobyMSI.msi.
[1633C:DAFC][2022-08-26T12:44:30]i000: Cache acquire updated: 100
[1633C:DAFC][2022-08-26T12:44:30]i000: Progress report
[1633C:DAFC][2022-08-26T12:44:30]i000: Cache completed
[92D4:ED9C][2022-08-26T12:44:30]i323: Registering package dependency provider: {1E243719-FDEC-432D-8279-ACE4825D1F7D}, version: 1.0.0.0, package: MobyInstallerId
[1633C:58E8][2022-08-26T12:44:30]i000: Execute package phase beginning
[92D4:ED9C][2022-08-26T12:44:30]i301: Applying execute package: MobyInstallerId, action: Install, path: C:\ProgramData\Package Cache\{1E243719-FDEC-432D-8279-ACE4825D1F7D}v1.0.0.0\MobyMSI.msi, arguments: ' ALLUSERS="1" ARPSYSTEMCOMPONENT="1" MSIFASTINSTALL="7"'
[1633C:58E8][2022-08-26T12:44:30]i000: Received MSI message: 1: 0 2: 1033 3: 1252 
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 0
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: 1: 0 2: 1033 3: 0 
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Message type: 0, Argument: 1033, 0
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Message type: 1, Argument: Moby
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Action 12:44:31: INSTALL. 
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Action 12:44:31: FindRelatedProducts. Searching for related applications Found application: [1]
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Action 12:44:31: LaunchConditions. Evaluating launch conditions
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Action 12:44:31: ValidateProductID. 
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Action 12:44:31: CostInitialize. Computing space requirements
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Action 12:44:31: FileCost. Computing space requirements
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Action 12:44:31: CostFinalize. Computing space requirements
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Action 12:44:31: MigrateFeatureStates. Migrating feature states from related applications Application: [1]
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Action 12:44:31: InstallValidate. Validating install
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 15
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 50
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 50
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Action 12:44:31: RemoveExistingProducts. Removing applications Application: [1], Command line: [2]
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Action 12:44:31: InstallInitialize. 
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Action 12:44:31: ProcessComponents. Updating component registration
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Action 12:44:31: GenerateScript. Generating script operations for action: [1]
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 51
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 52
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: 
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 54
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 55
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 57
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Action 12:44:31: UnpublishFeatures. Unpublishing Product Features Feature: [1]
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Action 12:44:31: RemoveFiles. Removing files File: [1], Directory: [9]
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Action 12:44:31: InstallFiles. Copying new files File: [1],  Directory: [9],  Size: [6]
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: 
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 58
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 59
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 61
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 62
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 64
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 65
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 66
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Action 12:44:31: RegisterUser. Registering user [1]
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Action 12:44:31: RegisterProduct. Registering product [1]
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: 
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 68
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 69
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 71
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 72
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 73
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Action 12:44:31: PublishFeatures. Publishing Product Features Feature: [1]
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: 
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 75
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 76
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Action 12:44:31: PublishProduct. Publishing product information
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: 
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 78
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 79
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 80
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 82
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 83
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 85
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 86
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 87
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Action 12:44:31: InstallFinalize. 
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 95
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Message type: 0, Argument: 1033, 0
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Message type: 1, Argument: Moby
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Action 12:44:31: ProcessComponents. Updating component registration
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 95
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 95
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 97
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Action 12:44:31: InstallFiles. Copying new files File: [1],  Directory: [9],  Size: [6]
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 97
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Action 12:44:31: RegisterProduct. Registering product [1]
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 99
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: 1: {1E243719-FDEC-432D-8279-ACE4825D1F7D} 
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Action 12:44:31: PublishFeatures. Publishing Product Features Feature: [1]
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: 1: ProductFeature 
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Action 12:44:31: PublishProduct. Publishing product information
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: 1: {1E243719-FDEC-432D-8279-ACE4825D1F7D} 
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Message type: 2, Argument: 0
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Action 12:44:31: RollbackCleanup. Removing backup files File: [1]
[1633C:58E8][2022-08-26T12:44:31]i000: Execute progress report: 100
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: Message type: 2, Argument: 1
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: 1: 2 2: 0 
[1633C:58E8][2022-08-26T12:44:31]i000: Received MSI message: 1: 2 2: 1 
[1633C:58E8][2022-08-26T12:44:31]i000: Progress report
[1633C:58E8][2022-08-26T12:44:31]i000: Execute package complete
[1633C:58E8][2022-08-26T12:44:31]i319: Applied execute package: MobyInstallerId, result: 0x0, restart: None
[92D4:ED9C][2022-08-26T12:44:31]i325: Registering dependency: {e1df1198-db62-4c5b-9bd9-afb39d021057} on package provider: {1E243719-FDEC-432D-8279-ACE4825D1F7D}, package: MobyInstallerId
[92D4:ED9C][2022-08-26T12:44:31]i372: Session end, registration key: SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{e1df1198-db62-4c5b-9bd9-afb39d021057}, resume: ARP, restart: None, disable resume: No
[92D4:ED9C][2022-08-26T12:44:31]i371: Updating session, registration key: SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{e1df1198-db62-4c5b-9bd9-afb39d021057}, resume: ARP, restart initiated: No, disable resume: No
[1633C:58E8][2022-08-26T12:44:31]i000: Apply phase complete
[1633C:58E8][2022-08-26T12:44:31]i399: Apply complete, result: 0x0, restart: None, ba requested restart:  No
[1633C:166C0][2022-08-26T12:44:34]i000: Into Process Install
[1633C:166C0][2022-08-26T12:44:34]i000: InstallLabel: Complete
```

I am using the OnDetectPackageComplete event to get the registration wizard panel initialized, so I know the events are firing as they should in my development setup. You can also see in the above log, that I have Engine.Log calls in each of the OnXXX events to show they are being executed.

So, why would one installer work and the very same installer on another machine not work...

I have to get this done pronto, so I'd appreciate a response and I thank you in advance for your help!!

Steve Miller

**Comment by robmen** [comment]
It is possible through discussion that a specific bug in the WiX Toolset will be uncovered, and then a bug with those details can be opened. But usually, someone can help you find the error in your code via the [support options](https://www.firegiant.com/support-options/).

**Comment by mobycorp** [comment]
Hi Rob,

Thank you so much for getting back to me!! I'm deeply grateful! And, point
well taken about getting some better support coverage...

And, yes, I thought I had plagiarized the WixBA project completely, but I
see there were several methods I failed to implement. I am in the process
of fixing all of that now, so hopefully, I will get beyond the barrier and
be on my way!!  BTW, I have written dozens of WiX installers, but this is
the first custom BA project I have ever done. It is a tad more
involved!!! 😇😀😁

Thanks again for your quick response!
Steve
